Cannot build DD4hep examples

With the current HEAD I cannot build the DD4hep examples.

 - OS version: e.g. centos7
 - Compiler version: e.g GCC 8.2
 - DD4hep version: 
        just after: https://github.com/AIDASoft/DD4hep/pull/736

- observed error:
```
[ 42%] Building CXX object DDDigi/CMakeFiles/DDDigiExampleLib.dir/src/DigiTestFalphaNoise.cpp.o
[ 42%] Built target ConditionsExample
[ 43%] Generating G__DDG4UserDict.cxx, ../lib/G__DDG4UserDict_rdict.pcm
error: unknown argument: '-rmf=/afs/desy.de/project/ilcsoft/sw/x86_64_gcc82_centos7/HEAD-2020-11-10/DD4hepExamples/HEAD/build/lib/G__DDG4UserDict.rootmap'
error: unknown argument: '-rml=libDDG4UserLib.so'
[ 44%] Building CXX object DDCMS/CMakeFiles/DDCMS.dir/src/plugins/DDTECCoolAlgo.cpp.o
error: unknown argument: '-rmf=/afs/desy.de/project/ilcsoft/sw/x86_64_gcc82_centos7/HEAD-2020-11-10/DD4hepExamples/HEAD/build/lib/G__DDG4UserDict.rootmap'
error: unknown argument: '-rml=libDDG4UserLib.so'
Error: Parsing Linkdef file /afs/desy.de/project/ilcsoft/sw/x86_64_gcc82_centos7/HEAD-2020-11-10/DD4hep/HEAD/include/ROOT/LinkDef.h
make[2]: *** [DDG4/G__DDG4UserDict.cxx] Error 1
make[1]: *** [DDG4/CMakeFiles/DDG4UserLib.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
```

As pointed out by @andresailer most likely the problem is due to this:

https://github.com/AIDASoft/DD4hep/pull/728/files#diff-31e44c0134fe48663d479a03db01ac9c8e60585265894f008a126a6615b1ab8b

- root version used:
/afs/desy.de/project/ilcsoft/sw/x86_64_gcc82_centos7/root/6.18.04

--- a/examples/DDG4/CMakeLists.txt
+++ b/examples/DDG4/CMakeLists.txt
@@ -30,7 +30,7 @@ if (DD4HEP_USE_GEANT4)
   SOURCES ${DD4hep_DIR}/include/ROOT/Warnings.h src/Dictionary.h
   LINKDEF ${DD4hep_DIR}/include/ROOT/LinkDef.h
   DEFINITIONS DD4HEP_DICTIONARY_MODE
-  OPTIONS -rmf=${LIBRARY_OUTPUT_PATH}/G__DDG4UserDict.rootmap -rml=libDDG4UserLib.so
+  OPTIONS -rmf ${LIBRARY_OUTPUT_PATH}/G__DDG4UserDict.rootmap -rml libDDG4UserLib.so
   OUTPUT ${LIBRARY_OUTPUT_PATH}
   )
   #----  Example of a client library with user defined plugins  --------------------

